test/ArrayField_test.js

```diff
@@ -793,6 +843,65 @@ describe("ArrayField", () => {
       expect(node.querySelector(".array-item-add button")).to.be.null;
     });
 
+    it("[fixed-noadditional] should not provide an add button regardless maxItems", () => {
+      const maxItemsSchema = Object.assign({ maxItems: 3 }, schema);
```

Comment:
We tend to prefer object spread in this codebase, so you could write `{ maxItems: 3, ...schema }`.